portfolio/src/main/webapp/src/com.js

```diff
@@ -253,7 +277,16 @@ function handleAddComment() {
  */
 function handleDeleteAllComments() {
   deleteAllCommentsFromServer()
-    .then(handleUpdateComments)
+    .then((response) => {
+      // Displays login error message.
+      if (response.status === 'no-login') {
+        document.getElementById('no-login-modal-message').innerHTML =
+          'You must be logged in to delete all comments.';
+        // Show message.
+        document.getElementById('no-login-modal').style.display = 'flex';
```

Comment:
Instead of setting styles here, you can style with `display: flex;` in the CSS file for the element and  toggle the [`hidden` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden).

Reply:
Will change in this PR.